dashboard/app/views/regional_partners/show.html.haml

```diff
@@ -84,6 +87,14 @@
 = link_to 'Back', regional_partners_path
 %br
 %h2 Regions
+%p
+  Region mapping for this partner can be replaced by uploading a CSV file that includes the header
+  %b Region,
+  where a region is a zip code or 2 letter state abbreviation.
+= form_tag({action: 'replace_mappings'}, multipart: true) do
+  = file_field_tag('regions', accept: 'text/csv', style: 'line-height: initial;')
```

Comment:
Do we limit size of the uploaded file? If someone upload a really big file, will it negatively impact our server performance?

Reply:
good question--as far as I can tell there are no limits built in, I will add a limit for sanity checking. The file should always be very small because it is a csv, and only users with edit regional partner permissions can access the page so the risk of a large upload is small.

Reply:
added both a client side and server side check that file is < 500 kb. Based that off the (unrealistic) case that someone wants to upload a CSV with every zip code in the US in it (about 42k), which would only be around 420 kb

Reply:
Sounds good! It's a good practice to define boundaries (values, size etc.) for user input data.